Mixpanel/InAppNotifications.swift

```diff
@@ -79,20 +82,32 @@ class InAppNotifications: NotificationViewControllerDelegate {
     }
 
     @discardableResult
-    func notificationShouldDismiss(controller: BaseNotificationViewController, callToActionURL: URL?) -> Bool {
+    func notificationShouldDismiss(controller: BaseNotificationViewController,
+                                   callToActionURL: URL?,
+                                   shouldTrack: Bool,
+                                   additionalTrackingProperties: Properties?) -> Bool {
         if currentlyShowingNotification?.ID != controller.notification.ID {
             return false
         }
 
         let completionBlock = {
+            if shouldTrack {
+                var properties = additionalTrackingProperties
+                if let urlString = callToActionURL?.absoluteString {
+                    if properties == nil {
+                        properties = [:]
+                    }
+                    properties!["url"] = urlString
```

Comment:
`if let properties = properties { properties["url"] = urlString }`

Reply:
i just tried this and it said i can't assign like that because properties is a let constant